### ISSUE TITLE
Vikram Singh Sikarwar: new volunteer via Mozilla Science Lab Collaborate

### ISSUE THREAD

**Issue by vsikarwar87** [JaredHawkins/TweetGeoViz#90]
Hi, I am interested in this project and have time to contribute as a developer and designer.
It could be a great tool to analyse people health based on geolocation/age and other factors.<p><br><blockquote>This issue was created by @vsikarwar87 via <a href='https://mozillascience.org/collaborate'>Mozilla Science Lab Collaborate</a></blockquote></p>

**Comment by anvk** [comment]
@vsikarwar87 hi. glad to hear you are interested in the project. With what technologies are you familiar with? We also would benefit from any designer skills as well. Could you summarize your skill set here for us to understand how you could possibly help us with the project.
Thanks!

**Comment by vsikarwar87** [comment]
Hi,

I am a Java Developer and can also code in Python / JavaScript.
i have some free time and would like to expand my horizon in new problems
Please help me in understanding roadmap and inside of project, so that I will start working on it.

**Comment by anvk** [comment]
@pydevhacker try to build the project first and see if it works for you. I'm in the middle of another branch for TweetGeoViz once it goes in I will see some tasks more clear.
